atomic_reactor/tasks/common.py

```diff
@@ -87,9 +91,12 @@ def _drop_known_unset_args(cls, args: dict) -> dict:
 class Task(abc.ABC, Generic[ParamsT]):
     """Task; the main execution unit in atomic-reactor."""
 
+    ignore_sigterm: ClassVar[bool] = False
+
     def __init__(self, params: ParamsT):
         """Initialize a Task."""
         self._params = params
+        self.fs_watcher = FSWatcher()
```

Comment:
Hm, maybe not all tasks should have a FSWatcher() because the data from it are used only in store_metadata plugin to export it in annotations.

Reply:
@chmeliik @tkdchen what do you think?

Reply:
Hmm, not gonna lie, I don't really know what `FSWatcher` is good for :smile: 

I guess it could be useful if we wanted to figure out how much disk space our builds are actually using, and for that, we would want to keep track of the usage across all tasks. Which would mean saving the results in context data somehow, and that sounds kinda complicated, so maybe it should be a separate ticket.

There might also be other options - e.g. just _log_ the usage after each task, which would make it easier to query in Splunk anyway. Or get rid of it entirely and rely on some OpenShift monitoring mechanism to get usage stats instead.

TL;DR, I think FSWatcher doesn't really need to do anything sensible at the moment and we should figure it out later